src/renderer/components/test-utils/get-application-builder.tsx

```diff
@@ -213,6 +218,35 @@ export const getApplicationBuilder = () => {
     },
   }));
 
+  mainDi.override(lensFetchInjectable, (di) => {
```

Comment:
Is this "lensFetchFake" supposed to work like `channel-fakes`, ie. with multiple environments/dis collaborating together? Shouldn't something called `lensFetchInjectable` be present only in renderer? Or does main `lensFetch` stuff also?

Reply:
Main `lensFetch`'s too and this fake is present in renderer. Continue reading.